docs/en/ingest-management/commands.asciidoc

```diff
@@ -521,6 +523,30 @@ elastic-agent inspect components log-default
 <hr>
 ++++
 
+[discrete]
+[[elastic-agent-privileged-command]]
+== elastic-agent privileged
+
+Run {agent} with full superuser privileges.
+This is the usual, default running mode for {agent}.
+The `privileged` command allows you to switch back to running an agent with full administrative privileges when you have been running it in `unprivileged` mode.
+
+Changing from `unprivileged` to `privileged` mode is supported only on Linux systems. 
```

Comment:
Switching between privileged and unprivileged works on all OS's.

Reply:
I've removed the restriction about being supported only on Linux.